README.md

```diff
@@ -87,13 +113,71 @@ Logs are printed to both render console and stdout. Logs are also written to a l
+- `enhancement`: the issue is an enhancement to either an existing feature in the product, or to the infrastructure around the development process of the product
+- `blocked`: the issue cannot be resolved as it depends on a fix in any of its dependencies
+- `good first issue`: an issue considered more accessible for any developer trying to start contributing
+
+### Commits and Pull Requests
+
+We use [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0-beta.3/) style messages. (More usually [with a scope](https://www.conventionalcommits.org/en/v1.0.0-beta.3/#commit-message-with-scope)) for commits.
+
+Commits should therefore strive to tackle one issue/feature, and code should be pre-linted before commit.
+
+PRs should clearly link to an issue to be tracked on the project board. A PR that implements/fixes an issue is linked using one of the [GitHub keywords](https://help.github.com/articles/closing-issues-using-keywords). Although these type of PRs will not be added themselves to a project board (just to avoid redundancy with the linked issue). However, PRs which were sent spontaneously and not linked to any existing issue will be added to the project and should go through the same process as any other tasks/issues.
+
+Where appropriate, commits should _always_ contain tests for the code in question.
+
+### Releases and Changelog
+
+The change log is currently maintained manually, each PR sent is expected to have the corresponding modification in the [CHANGELOG](CHANGELOG.MD) file, under the ['Unreleased' section](CHANGELOG.MD#unreleased). We are planning to start using [standard-version](https://www.npmjs.com/package/standard-version) shortly for maintaining the changelog.
+
+The release process is triggered by the maintainers of the package, by bumping the package version according to the [SemVer](https://semver.org/) spec, and pushing a tag to have our Travis CI scripts to automatically create the new version of `@maidsafe/safe-node-app` package and publish it at [https://www.npmjs.com/package/@maidsafe/safe-node-app](https://www.npmjs.com/package/@maidsafe/safe-node-app)
+
+### Copyrights
+
+Copyrights in the SAFE Network are retained by their contributors. No copyright assignment is required to contribute to this project.
+
 ## Further Help
 
 You can discuss development-related questions on the [SAFE Dev Forum](https://forum.safedev.org/).
-If you are just starting to develop an application for the SAFE Network, it's very advisable to visit the [SAFE Network Dev Hub](https://hub.safedev.org) where you will find a lot of relevant information, including tutorials.
+If you are just starting to develop an application for the SAFE Network, it's very advisable to visit the [SAFE Network Dev Hub](https://hub.safedev.org) where you will find a lot of relevant information, including a [tutorial to create an example SAFE desktop application](https://hub.safedev.org/platform/nodejs) which makes use of this package.
```

Comment:
@joshuef  how about something like `a [tutorial to create an example SAFE web application](https://hub.safedev.org/platform/web/) which makes use of the API exposed by the SAFE browser.` ?